interfaces/cython/cantera/examples/thermo/isentropic_units.py

```diff
@@ -0,0 +1,65 @@
+"""
+Isentropic, adiabatic flow example - calculate area ratio vs. Mach number curve
+
+Requires: cantera >= 2.5.0, matplotlib >= 2.0
```

Comment:
Can you please update these versions here? The Cantera version needs to be updated to 2.6.0, which is when this feature will be available. Similar for the other examples.

This particular example can also delete the `matplotlib` requirement, since you deleted that.